analyzers/tests/SonarAnalyzer.UnitTest/Rules/ClearTextProtocolsAreSensitiveTest.cs

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
Maybe a   Verifier.VerifyNoIssueReported test case can be added to verify that the hotspot configuration checking is done correctly.

Reply:
Good catch! I forgot about this.